### PR TITLE
[mysql][hotfix] serialize isSuspended flag for MySqlBinlogSplit

### PR DESCRIPTION
**Pull request by zhjwpku**
Signed-off-by: 元组 <zhaojunwang.zjw@alibaba-inc.com>

### CODE REVIEW

File: flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/enumerator/MySqlSourceEnumerator.java [hunk 1]
```diff
@@ -85,12 +87,17 @@ public MySqlSourceEnumerator(
         this.sourceConfig = sourceConfig;
         this.splitAssigner = splitAssigner;
         this.readersAwaitingSplit = new TreeSet<>();
+        if (splitAssigner.getAssignerStatus() == NEWLY_ADDED_ASSIGNING
```
Comment (leonardBang) [inline flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/enumerator/MySqlSourceEnumerator.java:90]: We can use `AssignerStatus.isAssigning(splitAssigner) || AssignerStatus.isAssigningFinished(splitAssigner)` to cover newly assigning and initial assigning.

**Review by leonardBang — APPROVED** [review]
Thanks @zhjwpku for the update, LGTM

**Comment by leonardBang** [comment]
@zhjwpku Could you also open a PR for release-2.2 branch?

**Comment by zhjwpku** [comment]
> 

yeah, see https://github.com/ververica/flink-cdc-connectors/pull/1055

**Pull request merged**